### DISCUSSION TITLE
EPUB 3 Media Overlays versus TTS User Settings

### DISCUSSION THREAD

**Discussion by MyDK** [edrlab/thorium-reader#1774] [Ideas]
In an EPUB 3 publication with prerecorded audio files (embedded media), how will Thorium handle if the User should choose to overrule the default settings, and instead choose to have text read aloud as synthetic speech (TTS)?

**Comment by danielweck** [comment]
Dear Marianne, this is a great suggestion. I have explored this question too when implementing the common audio toolbar at the top of the UI, for both TTS and EPUB3 Media Overlays (mutually-exclusive, depending on whether a publication has MO or not).
I wonder what modifications to the UI would be suitable, without overloading the already-dense controls layout. I am thinking about adding a configuration checkbox in the "readaloud" section, which is not intrusive and perfect placement for an "unusual" / "not used very often" feature. Conversely, if the feature is used often, the affordance must be placed prominently in the UI, for example as first-class citizen in the top audio toolbar. What do you think?

**Comment by ways2read** [comment]
Easyreader for desktop has the option "Use book audio when available" in the voice options menu. It is checked by default.

**Comment by MyDK** [comment]
@danielweck - I am ‘just’ a content creator, not app developer, so I don’t know enough about how RS/AT works to give you a proper answer. But I am a huge fan of multiple User Options in Reading situations (lesser fan in input-content-for-convertion-situations)
But functionality comes first.

**Comment by Moonbase59** [comment]
I was happy to find that Thorium actually supports EPUB3 with (audio) media overlays—hooray! I’d go with @ways2read’s suggestion of having a "use book audio when available" check button.
